mhep/config/settings/base.py

```diff
@@ -275,3 +275,13 @@
 CORS_ORIGIN_WHITELIST = ["http://localhost:9091"]
 CORS_ALLOW_METHODS = list(default_methods)
 CORS_ALLOW_HEADERS = list(default_headers)
+
+REST_FRAMEWORK = {
+    'DEFAULT_AUTHENTICATION_CLASSES': ([
+        'rest_framework.authentication.SessionAuthentication',
+        'rest_framework.authentication.BasicAuthentication',
+    ]),
+    'DEFAILT_PERMISSION_CLASSES': (
```

Comment:
typo

Reply:
I'm not sure it's useful to specify `DEFAULT_PERMISSION_CLASSES` any how - what's the intention?

Reply:
Hah - the typo meant it wasn't being applied, so having no effect. My thinking was the API was only used by logged in users, so that should be the default. Maybe it's something we introduce later so as not to introduce views that are publically accessible by mistake?